docs/pipelines/access-control/index.md

```diff
@@ -0,0 +1,29 @@
+# Access control
+
+Gruntwork Pipelines grants permissions by defining three [GitHub Teams](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams), which should map to three separate personas in your organization. Each team and its permissions are designed to grant the _least possible permissions_ to each individual (or machine user) in your organization for them to be able to perform changes to your infrastructure.
+
+### GitHub Team: infrastructure-administrators
+
+The `infrastructure-administrators` team is for engineers who likely work on the IaC codebase daily, and  have administrative AWS permissions. These users will also have permission to both read and write Pipelines configuration files, and the GitHub Actions Workflow files that are used to trigger Pipelines.
+
+### GitHub Team: infrastructure-collaborators
+
+The `infrastructure-collaborators` team is for engineers who work on the IaC codebase daily but who _do not_ have administrative permissions in AWS. The users will also have permission to read Pipelines configurations files, but not to write (or modify) them.
+
+### Github Team: ci-code-read-only
+
+ The `ci-code-read-only` team is meant for a single machine user that can read your `infrastructure-live` and `infrastructure-modules` (a repository where you can define custom OpenTF (Terraform) modules for your organization) repos.
+
+### Visualizing all three teams
```

Comment:
The header and intro sentence here both seem unnecessary. It would be fine to just drop the image directly into the page and add a caption to call out what it illustrates.

Reply:
sounds good

Reply:
Updated in [d174cc4](https://github.com/gruntwork-io/docs/pull/1099/commits/d174cc4ec5c3cd361a0fcc6da399d736fd7fb803)